src/TMap.cpp

```diff
@@ -2584,3 +2584,10 @@ void TMap::update()
     }
 }
 
+QString TMapLabel::base64EncodePixmap() {
```

Comment:
:information_source: FYI Our code style is to put the opening brace of a function/method on the next line... :wink:

Reply:
Ahhh yeah, usually remember it.